monitoring/api/v3/alerts-client/snippets_test.py

```diff
@@ -17,6 +17,7 @@
 import random
 import string
 
+from gcp_devrel.testing import eventually_consistent
```

Comment:
In general I'd prefer we stop relying on this module altogether. It's not maintained and a lot of the logic isn't really great for testing. 

IIRC, `eventually_consistent` just attempts the function until it passes. This could make the issue of failures during concurrent tests worse. There are other libraries (like [flaky](https://pypi.org/project/flaky/)) that do this better and don't add extra maintenance on us.

Reply:
@kurtisvg
Flaky sounds like a plan. I'll use flaky for this test.

But, I kinda disagree with your two points; for the first point "It's not maintained", we can maintain it, right? I'm happy to do that from now on. For the second point, "lot of the logic isn't really great for testing." Can you elaborate? Maybe you can file a bug on the [repo](https://github.com/GoogleCloudPlatform/python-repo-tools)?

Reply:
I still think `eventually_consistent` is good for eventually consistent tests. It is basically a wrapper around retrying module with default exponential backoff parameters.

But I agree that flaky is more suitable with this test.

Reply:
RE: maintenance - it doesn't seem worth the effort of us maintaining and releasing a special module just for testing our samples. Are our libraries really so hard to test that we can't find an out of the box solution to reach reasonable tests? (I think the answer is no)

RE: logic - IMO, flakey tests are bad tests. Running a test several times hoping at least one passes means that potentially, our users also have to run the sample multiple times to make sure that it's working. Ideally, the snippet should work the first time, every time.

Reply:
@kurtisvg Thanks for the discussion. Let's discuss further off PR :)